docs/es-overview.asciidoc

```diff
@@ -2,150 +2,122 @@
 [chapter, role="xpack"]
 = Elastic Security overview
 
-Elastic Security combines the threat detection of SIEM and the prevention, detection, and response of Endpoint Security via an autonomous agent into a unified software solution. These analytical and protection capabilities, leveraged by the speed and extensibility of Elasticsearch, enable analysts to defend their organization from threats before damage and loss occur.
+Elastic Security combines SIEM threat detection features with endpoint
```

Comment:
Are we still allowed to use the word SIEM?

Reply:
Yes 😄

Reply:
Unless @MikePaquette says otherwise...